src/services/report.ts

```diff
@@ -133,3 +133,16 @@ export async function getReportDetail(reportId: string): Promise<Report> {
 
   return deserializedReport(res.data);
 }
+
+export async function assignAdmin(reportId: string, adminId: number) {
+  const url = `/reports/${reportId}/assign/`;
+  devPrint('called');
```

Comment:
might want to remove

Reply:
btw best to reply if you choose not to

Reply:
I thought I removed it? Let me double check

Reply:
Oh I removed some other one I think. My bad.